.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  trailingComma: 'all',
+  tabWidth: 2,
+  semi: false,
+  singleQuote: true,
+  arrowParens: 'always',
+}
```

Comment:
I am hesitant to modify the code styles we [already have in place](https://github.com/nih-sparc/sparc-app/blob/master/.eslintrc.js). We are already following `prettier/recommended` which has all of these except `trailingComma: 'all'`.